Formula/ponyc.rb

```diff
@@ -42,6 +42,9 @@ def install
   end
 
   test do
+    # ENV.cc returns llvm_clang, which does not work in a test block.
+    ENV["CC"] = Formula["llvm"].opt_bin/"clang"
```

Comment:
Probably want to call `ENV.clang` here, actually, since presumably this path does not always exist on macOS. (Would be nice to get this formula to stop vendoring LLVM.)

Reply:
On the topic of vendored LLVM, also note that I encountered issues with it on the latest bump, as notified here: https://github.com/Homebrew/homebrew-core/pull/124342#issuecomment-1446795198

Don't know if others could reproduce it, I guess it'll need to wait for the next bump?

Reply:
> On the topic of vendored LLVM, also note that I encountered issues with it on the latest bump, as notified here: [#124342 (comment)](https://github.com/Homebrew/homebrew-core/pull/124342#issuecomment-1446795198)
> 
> Don't know if others could reproduce it, I guess it'll need to wait for the next bump?

Couldn't reproduce it:

```
$ brew fetch -svd ponyc
==> Cloning https://github.com/ponylang/ponyc.git
Cloning into '/home/linuxbrew/.cache/Homebrew/ponyc--git'...
==> Checking out tag 0.54.0
HEAD is now at 2704bc0c Update VERSION in preparation for 0.54.0 release
Submodule 'lib/llvm/src' (https://github.com/llvm/llvm-project) registered for path 'lib/llvm/src'
Cloning into '/home/linuxbrew/.cache/Homebrew/ponyc--git/lib/llvm/src'...
Submodule path 'lib/llvm/src': checked out '1f9140064dfbfb0bbda8e51306ea51080b2f7aac'
/home/linuxbrew/.cache/Homebrew/ponyc--git/lib/llvm/src
```

This looks like an issue with your local system configs (Git, `ulimit`, etc.), though; https://github.com/llvm/llvm-project is an exceptionally large repo. I guess a manually clone will likely fail, too. https://stackoverflow.com/questions/21277806/fatal-early-eof-fatal-index-pack-failed